common/dynamicconfig/dynamicproperties/constants.go

```diff
@@ -2524,6 +2524,13 @@ const (
 	// Value type: Duration
 	// Default value: 3 seconds
 	GlobalRatelimiterUpdateInterval
+	// FrontendMaxWorkerPollWait is the maximum duration a worker poll request (PollForActivityTask, PollForDecisionTask)
+	// will wait for a rate limit token before being rejected
+	// KeyName: frontend.maxWorkerPollWait
```

Comment:
Naming things is hard, and I don't have a great name for this, but I think this name may be confusing. Worker poll requests are expected to wait for quite some time (see `matching.longPollExpirationInterval`, they open the connection for up to a minute by default ), and this doesn't impact that behavior.

Maybe `frontend.maxWorkerPollDelay` better describes it? I'm honestly not sure.